Add new BBE for ballerina/file module (#18575)
ballerina-platform/ballerina-lang: stdlib/file/src/main/ballerina/src/file/Module.md

    string|error results = file:createDir(“foo/bar”);
    

>     // Create a directory with any non existence parents.

Reviewer: ```suggestion
    // Create a directory with any none-existent parents.
```

>     string|error results = file:createDir(“foo/bar”, true);

Reviewer 2: We should use some `io:println() lines to validate the results of each of these operations. They should appear in .out file

Reviewer 2: sorry this comment applies to .bal file.

>     // Remove file or directory in specified file path.

Reviewer: ```suggestion
    // Remove the file or directory in the specified file path.
```

    error? results = file:remove(“foo/bar.txt”);

>     // Remove directory in specified file path with all children.

Reviewer: ```suggestion
    // Remove the directory in the specified file path with all its children.
```

    error? results = file:remove(“foo/bar”, true);
    

>     // Rename(Move) file or directory to new path.

Reviewer: ```suggestion
    // Rename(Move) the file or directory to the new path.
```

    string|error results = file:createFile(“bar.txt”);
    
    // Get metadata information of the file.
    file:FileInfo|error result = file:getFileInfo(“foo/bar.txt”);
    

>     // Get list of files in the directory.

Reviewer: ```suggestion
    // Get the list of files in the directory.
```

    file:FileInfo[]|error results = file:readDir(“foo/bar”);
    

>     // Copy file ot directory to new path.

Reviewer: ```suggestion
    // Copy the file or directory to the new path.
```
